### PR TITLE
Fix: fix text key encoding

### PR DESCRIPTION
**Pull request by aliel**

### CODE REVIEW

File: src/aleph_client/__main__.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def _load_account(
 
     if private_key_str:
         if not private_key_file:
-            return ETHAccount(private_key_str.encode())
+            return ETHAccount(private_key_str)
```
Comment (odesenfans) [inline src/aleph_client/__main__.py:60]: Any particular reason for this change? Diving into the code, it seems like you can feed it bytes or strings, it doesn't really matter. I'd just like to understand why you want to change this.

Comment (aliel) [reply src/aleph_client/__main__.py:60]: It was a confusing with another related bug that fixed after this one (https://github.com/aleph-im/aleph-client/pull/67)
but another behavior still there when we provide a private-key as text using the CLI

Comment (hoh) [reply src/aleph_client/__main__.py:60]: OK, thanks !

Comment (aliel) [reply src/aleph_client/__main__.py:60]: regarding the doc => https://eth-account.readthedocs.io/en/stable/eth_account.html#module-eth_account.account
the method accept only key as hex str or bytes not encoded string

File: src/aleph_client/__main__.py [hunk 2]
```diff
@@ -56,10 +56,10 @@ def _load_account(
     """Load private key from a file"""
 
     if private_key_str:
-        if not private_key_file:
-            return ETHAccount(private_key_str.encode())
+        if private_key_str[:2] == '0x':
+            return ETHAccount(bytes.fromhex(private_key_str[2:]))
         else:
-            raise ValueError("Choose between a private key or a file, not both")
+            return ETHAccount(bytes.fromhex(private_key_str))
```
Comment (odesenfans) [inline src/aleph_client/__main__.py:62]:
```suggestion
if private_key_str.startswith("0x"):
    private_key_str = private_key_str[2:]

return EthAccount(bytes.fromhex(private_key_str))
```

**Review by odesenfans — COMMENTED** [review]
A small suggestion to remove a bit of duplication.

**Comment by hoh** [comment]
Mypy fails with the error `/opt/aleph-client/src/aleph_client/__main__.py:66: error: Name "EthAccount" is not defined`

**Pull request merged**